kratos/python/add_properties_to_python.cpp

```diff
@@ -168,18 +188,24 @@ void  AddPropertiesToPython(pybind11::module& m)
     .def("SetValue", SetValueHelperFunction1< Properties, Variable< ConstitutiveLawBaseType::Pointer > >)
     .def("GetValue", GetValueHelperFunction1< Properties, Variable< ConstitutiveLawBaseType::Pointer > >)
 
-	.def("GetTable", GetTableHelperFunction1< Properties, Variable< double > , Variable<double> >, return_value_policy::reference_internal)
+    .def("GetTable", GetTableHelperFunction1< Properties, Variable< double > , Variable<double> >, return_value_policy::reference_internal)
     .def("GetTable", GetTableHelperFunction1< Properties, VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > , Variable<double> >, return_value_policy::reference_internal)
     .def("GetTable", GetTableHelperFunction1< Properties, Variable<double>, VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > >, return_value_policy::reference_internal)
     .def("GetTable", GetTableHelperFunction1< Properties, VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > , VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > >, return_value_policy::reference_internal)
     .def("SetTable", SetTableHelperFunction1< Properties, Variable< double > , Variable<double> >)
     .def("SetTable", SetTableHelperFunction1< Properties, VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > , Variable<double> >)
     .def("SetTable", SetTableHelperFunction1< Properties, Variable<double>, VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > >)
     .def("SetTable", SetTableHelperFunction1< Properties, VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > , VariableComponent<VectorComponentAdaptor<array_1d<double, 3> > > >)
-	.def("__repr__", &Properties::Info) //self_ns::str(self))
+    .def("__repr__", &Properties::Info) //self_ns::str(self))
```

Comment:
there was a discussion some time back on how to improve this to have propoer output.

Reply:
In this case I just added a trabulation

Reply:
just do

      .def("__str__", PrintObject<Properties>)

Reply:
actually this exists already a few lines below, i.e. please remove this line